doc/template/HOW-TO.md

```diff
@@ -0,0 +1,128 @@
+            -o $@ $<
+    +       sed -i 's/^<img /<img class="img-responsive" /g' $@
+
+     # Pattern to convert R Markdown to Markdown.
+     %.md: %.Rmd $(R_CHUNK_OPTS)
+
+To avoid losing my work, I save it in the staging area:
+
+    $ git add -u
+
+Now I compile the HTML files:
+
+    $ make -B preview
+
+And verify the changes.
+
+    $ git diff
+    diff --git a/01-one.html b/01-one.html
+    index 7622e55..1fc0d45 100644
+    --- a/01-one.html
+    +++ b/01-one.html
+    @@ -56,7 +56,8 @@ program</code></pre>
+     </div>
+     <p>or an image from the <code>figs</code> directory:</p>
+     <div class="figure">
+    -<img src="fig/example.svg" title="this is the image&#39;s alt text" alt="this is the image&#39;s title" /><p class="caption">this is the image's title</p>
+    +<img class="img-responsive" src="fig/example.svg" title="this is the image&#39;s alt text" alt="this is the image&#39;s title" />
+    +<p class="caption">this is the image's title</p>
+     </div>
+     <div id="challenge-title" class="challenge panel panel-success">
+     <div class="panel-heading">
+
+This is exactly what I want. Great!!! I can now commit my changes.
+
+    $ git commit -m 'Add img-responsive class to img'
+    [example-centering-image a9cc86d] Add img-responsive class to img
+     1 file changed, 1 insertion(+)
+
+I will also save the change to the HTML file.
+
+    $ git add -u
+    $ git commit -m 'Update HTML'
+    [example-centering-image 1ebf8c3] Update HTML
+     1 file changed, 2 insertions(+), 1 deletion(-)
+
+## Copying the change to template
+
+If you have taken the Git lesson at a Software Carpentry workshop, you should be comfortable with what you see until now.
+What I need to do is copy my change at `Makefile` to [template](https://github.com/swcarpentry/lesson-template) using `git cherry-pick`. If you are trying to remember it from our workshop, don't panic because we don't teach it but you will discovery that it is very simple to use.
+
+Before anything else I need to know the ID of the commit I want to copy, the one where I changed `Makefile`.
+
+    $ git log --oneline -3
+    1ebf8c3 Update HTML
+    a9cc86d Add img-responsive class to img
+    5426365 Merge pull request #1 from gvwilson/updating-docs
+
+The commit that I want is `a9cc86d`. I will save the ID for a moment until I create a new branch starting from the template (**instead of the example**):
+
+    $ git checkout -b template-centering-image template-upstream/gh-pages
+
+Now I can "copy and paste" my change, using the commit ID, and for that I use
+
+    $ git cherry-pick a9cc86d
+    [template-centering-image 2004c4f] Add img-responsive class to img
+     Date: Sat Apr 4 18:19:26 2015 -0300
+     1 file changed, 1 insertion(+)
+
+I can check that the commit was copied
+
+    $ git log --oneline -2
+    2004c4f Add img-responsive class to img
+    cf25427 Merge pull request #218 from wking/trailing-newlines
+
+The commit ID change but this is OK and is desired.
```

Comment:
“change” → “changed”.  And I'd suggest “The commit ID changed because the new commit has a different parent than a9cc86d.” to be less handwavy about this sort of thing.  A few other things will have changed to (the root tree for the commit and the committer date), but I don't think it's worth mentioning those.